src/metatensor/models/experimental/soap_bpnn/tests/test_torchscript.py

```diff
@@ -18,7 +21,44 @@ def test_torchscript():
         },
     )
     soap_bpnn = Model(capabilities, DEFAULT_HYPERS["model"])
-    torch.jit.script(soap_bpnn, {"energy": soap_bpnn.capabilities.outputs["energy"]})
+    soap_bpnn = torch.jit.script(soap_bpnn)
+
+    system = ase.Atoms(
+        "OHCN",
+        positions=[[0.0, 0.0, 0.0], [0.0, 0.0, 1.0], [0.0, 0.0, 2.0], [0.0, 0.0, 3.0]],
+    )
```

Comment:
Why not 

```
system = System(
   species=torch.tensor([8, 1, 6, 7],
   positions=torch.tensor([[0.0, 0.0, 0.0], [0.0, 0.0, 1.0], [0.0, 0.0, 2.0], [0.0, 0.0, 3.0]])
   cell=torch.zeros((3, 3)),
)
```

Reply:
To me it looks fine since all the other tests also take in systems from `ase.Atoms`, but if it's a matter of dropping more dependencies for the test, then I'm for it.

Reply:
Opened issue